modules/gitops-creating-a-new-client-in-dex.adoc

```diff
@@ -0,0 +1,31 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+= Enabling the Dex openshift OAuth Connector
+
+Dex uses the users and groups defined within OpenShift by querying the platform provided `OAuth` server.
+
+. The `openShiftOAuth` property is used to trigger the Operator to auto configure the built-in OpenShift `OAuth` server.
+. The `groups` property is used to mandate the users to be part of one or all the groups in the groups list.
+. The `RBAC Policy` property is used to give the admin role in the Argo CD cluster to users in the OpenShift cluster-admins group.
```

Comment:
`Use the `RBAC Policy` to give the admin role in the Argo CD cluster to users in the OpenShift cluster-admins group.`